meta/template/string.lua

```diff
@@ -27,7 +27,7 @@ function string.char(byte, ...) end
 function string.dump(f, strip) end
 
 ---#DES 'string.find'
----@param s       string
+---@param s       string|number
 ---@param pattern string
```

Comment:
I believe the pattern _could_ also be a number
```suggestion
---@param pattern string|number
```

Reply:
Indeed it can.